docs/includes/ksql-includes.rst

```diff
@@ -333,3 +333,17 @@ To enable JMX metrics, set ``JMX_PORT`` before starting the KSQL server:
 
     $ export JMX_PORT=1099 && \
       <path-to-confluent>/bin/ksql-server-start <path-to-confluent>/etc/ksql/ksql-server.properties
+
+.. log limitations
```

Comment:
@apurvam : This change should not have gone in.  I suppose that was caused by re-branching this PR.  The text looks fine to me, but it should not have been part of this PR.  Also, a word of caution: Updating `includes/ksql-includes.rst` means that we must double-check other files in our docs that are pulling in snippets from `ksql-includes.rst` based on start/end line numbers (@joel-hamill knows more) -- so changes to `ksql-includes.rst` are likely to break existing parts of the docs and thus need double-checking.

Reply:
Second thought: Probably we are ok with this particular change plus the other changes below (docs/installation/installing.rst, docs/tutorials/basics-docker.rst, docs/tutorials/clickstream-docker.rst) etc. because we need them, as long as they are correct, in `4.1.0-post`.  (Technically they just are unrelated to this PR.)